src/store/record.js

```diff
@@ -94,6 +95,9 @@ let recordStore = {
         resetRegistry(state){
             state.sections.generalInformation.data.type = "";
         },
+        setCreatingNewRecord(state){
```

Comment:
Don't you also need a method to set it back to false when users are editing a record after creating a new record ?

Reply:
Good point, I'll have to look into that.